blog/config.toml

```
@@ -7,6 +7,7 @@ generate_rss = true
 
 languages = [
     { code = "zh-CN" }, # Chinese (simplified)
+    { code = "zh-tw" }, # Chinese (tradictional))
```

```suggestion
    { code = "zh-TW" }, # Chinese (tradictional))
```

I think the `TW` should be uppercase, see https://en.wikipedia.org/wiki/Zh-TW.